### PR TITLE
direct to MDN for information about JSON

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

File: locale/pt-br/knowledge/javascript-conventions/what-is-json.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ difficulty: 5
 layout: knowledge-post.hbs
 ---
 
+This page is not actively maintained and may be outdated. Please refer to [MDN's JSON page](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON) instead.
```
Comment (Trott) [inline locale/pt-br/knowledge/javascript-conventions/what-is-json.md]: @nodejs/nodejs-pt If someone can offer a translation for these two sentences, that would be great.

**Comment by Trott** [comment]
@nodejs/website Anyone up for reviewing this?

File: locale/pt-br/knowledge/javascript-conventions/what-is-json.md (hunk 1 above)
Comment (TiagoDanin) [reply locale/pt-br/knowledge/javascript-conventions/what-is-json.md]: @Trott I can help with this
```suggestion
Esta página não é mantida ativamente e talvez esteja desatualizada. Consulte a [página do MDN sobre JSON](https://developer.mozilla.org/pt-BR/docs/Learn/JavaScript/Objects/JSON) em vez disso.
```

File: locale/pt-br/knowledge/javascript-conventions/what-is-json.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ difficulty: 5
...
+
+<details>
+<summary>Original contents</summary>
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/javascript-conventions/what-is-json.md]:
```suggestion
<summary>Conteúdo original</summary>
```

**Review by nschonni — APPROVED** [review]

File: locale/pt-br/knowledge/javascript-conventions/what-is-json.md (hunk 1 above)
Comment (Trott) [reply locale/pt-br/knowledge/javascript-conventions/what-is-json.md]: Thank you very much!

**Pull request merged**